internal/dag/builder_test.go

```diff
@@ -11388,6 +11388,9 @@ func TestDAGInsert(t *testing.T) {
 
 	for name, tc := range tests {
 		t.Run(name, func(t *testing.T) {
+			if name != "insert httpproxy w/ infinite timeoutpolicy" {
+				return
+			}
```

Comment:
guessing this is left over from debugging :)

Reply:
Sorry, my bad! Glad you spotted it, now fixed 😊